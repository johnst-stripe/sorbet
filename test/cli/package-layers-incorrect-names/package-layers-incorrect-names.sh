cd test/cli/package-layers-incorrect-names || exit 1

../../../main/sorbet --silence-dev-message --stripe-packages --stripe-packages-layers=good_layer_name --max-threads=0 . 2>&1
