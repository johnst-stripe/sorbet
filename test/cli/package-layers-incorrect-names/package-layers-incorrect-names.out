another-bad-name/__package.rb:4: Argument to `layer` is `another_bad_name` but must be a valid layer name (good_layer_name) https://srb.help/3717
     4 |  layer 'another_bad_name'
                ^^^^^^^^^^^^^^^^^^

bad-layer-name/__package.rb:4: Argument to `layer` is `bad_layer_name` but must be a valid layer name (good_layer_name) https://srb.help/3717
     4 |  layer 'bad_layer_name'
                ^^^^^^^^^^^^^^^^
Errors: 2
